examples/base-path-and-path/tests/base_path_and_path_test.bal

```diff
@@ -24,11 +24,11 @@ function testFunc() {
     // Send a `GET` request to the specified endpoint.
     var response = httpEndpoint->post("/foo/bar", req);
     match response {
-        http:Response resp => {
+        var resp if resp is http:Response => {
```

Comment:
Shall we remote match and use if check with if (response is http:Response) etc?

Reply:
Adding sample code:
```ballerina
if (response is http:Response) {
    var jsonRes = check response.getJsonPayload();
    test:assertEquals(jsonRes, payload);
} else {
    test:assertFail(msg = "Failed to call the endpoint:");
}
```

Reply:
will add this change